.gitignore

```diff
@@ -247,3 +247,4 @@ test/k4FWCoreTest/**/*.root
 test/inputFiles/*.slcio
 test/gaudi_opts/testConverterConstants.py
 
+/tests/inputFiles/
```

Comment:
I'll add this to the `.gitignore` in https://github.com/key4hep/key4hep-dev-utils/tree/main/defaults, otherwise it will be overwritten if I do another sync

Reply:
Removed.